libs/devops/src/lib/firestoreMigrations/0079.ts

```diff
@@ -71,8 +65,8 @@ async function migrateNotifications(db: Firestore) {
   return runChunks(notifications.docs, async (doc) => {
     const notificationBase = doc.data() as Notification;
 
-    // Skip all notifications without organization or organization.denomination
-    if (!notificationBase?.organization || !(notificationBase.organization as any)?.denomination) return false;
+    // Skip all notifications without organization
+    if (!notificationBase?.organization) return false;
```

Comment:
I think you should keep the name check
```suggestion
    // Skip all notifications without organization or organization name
    if (!notificationBase?.organization?.name) return false;
```
That is probably why denomination was checked for previously too.